provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -1692,6 +1692,26 @@ func nodeGroupProperties(cluster, v2 bool) map[string]schema.PropertySpec {
 			TypeSpec:    schema.TypeSpec{Type: "integer"},
 			Description: "The size in GiB of a cluster node's root volume. Defaults to 20.",
 		},
+		"nodeRootVolumeDeleteOnTermination": {
```

Comment:
This is a bit meta, but I'm wondering what we can do to prevent something like this from happening again in the future. We've already increased the test coverage in the last weeks, but I feel like this is gonna happen again unless we codegen the node SDK as well

Reply:
https://github.com/pulumi/pulumi-eks/issues/1184 I didn't find a tracking issue so added one. This doesn't sound easy but sounds like something we definitely need to do.

Reply:
Ah you were quicker than me! Thanks for creating it